src/components/Reports/PlantingSitesSpeciesCellRenderer.tsx

```diff
@@ -47,6 +47,28 @@ export default function PlantingSiteSpeciesCellRenderer({ editMode, validate }:
       }
     };
 
+    if (column.key === 'growthForm') {
```

Comment:
move the mapping into the types as a util?
example: https://github.com/terraware/terraware-web/blob/main/src/types/Accession.ts#L33
in case it helps with reuse later

Reply:
Changes look good, thanks!!